app/configurator/components/ui-helpers.ts

```diff
@@ -726,60 +767,66 @@ export const getColorInterpolator = (
   }
 };
 
-export const categoricalPalettes: Array<{
-  label: string;
-  value: string;
-  colors: ReadonlyArray<string>;
-}> = [
-  {
-    label: "category10",
-    value: "category10",
-    colors: getPalette("category10"),
-  },
-  { label: "accent", value: "accent", colors: getPalette("accent") },
-  { label: "dark2", value: "dark2", colors: getPalette("dark2") },
-  { label: "paired", value: "paired", colors: getPalette("paired") },
-  { label: "pastel1", value: "pastel1", colors: getPalette("pastel1") },
-  { label: "pastel2", value: "pastel2", colors: getPalette("pastel2") },
-  { label: "set1", value: "set1", colors: getPalette("set1") },
-  { label: "set2", value: "set2", colors: getPalette("set2") },
-  { label: "set3", value: "set3", colors: getPalette("set3") },
-];
+const steppedPaletteSteps = [0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1];
 
-export const getDefaultCategoricalPalette = () => categoricalPalettes[0];
+const divergingPaletteKeys = [
+  "BrBG",
+  "PRGn",
+  "PiYG",
+  "PuOr",
+] as DivergingPaletteType[];
 
-const sequentialPaletteSteps = [
-  0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1,
-];
-// Diverging color palettes
-export const sequentialPalettes: Array<{
+export const divergingPalettes = divergingPaletteKeys.map((d) => ({
+  label: d,
+  value: d,
+  interpolator: getColorInterpolator(d),
+})) as {
+  label: string;
+  value: DivergingPaletteType;
+  interpolator: (t: number) => string;
+}[];
+
+export const divergingSteppedPalettes = divergingPaletteKeys.map((d) => ({
+  label: d,
+  value: d,
+  colors: steppedPaletteSteps.map((s) => getColorInterpolator(d)(s)),
+})) as {
   label: string;
   value: string;
   colors: ReadonlyArray<string>;
-}> = [
-  {
-    label: "BrBG",
-    value: "BrBG",
-    colors: sequentialPaletteSteps.map((d) => getColorInterpolator("BrBG")(d)),
-  },
-  {
-    label: "PRGn",
-    value: "PRGn",
-    colors: sequentialPaletteSteps.map((d) => getColorInterpolator("PRGn")(d)),
-  },
-  {
-    label: "PiYG",
-    value: "PiYG",
-    colors: sequentialPaletteSteps.map((d) => getColorInterpolator("PiYG")(d)),
-  },
-  {
-    label: "PuOr",
-    value: "PuOr",
-    colors: sequentialPaletteSteps.map((d) => getColorInterpolator("PuOr")(d)),
-  },
-];
-
-export const getDefaultSequentialPalette = () => sequentialPalettes[0];
+}[];
+
+export const getDefaultDivergingSteppedPalette = () =>
+  divergingSteppedPalettes[0];
+
+const sequentialPaletteKeys = [
+  "blues",
+  "greens",
+  "greys",
+  "oranges",
+  "purples",
+  "reds",
+] as SequentialPaletteType[];
+
+export const sequentialPalettes = sequentialPaletteKeys.map((d) => ({
+  label: d,
+  value: d,
+  interpolator: getColorInterpolator(d),
+})) as {
+  label: string;
+  value: SequentialPaletteType;
+  interpolator: (t: number) => string;
+}[];
```

Comment:
Nit: Why do we have to explicitly type here ? I think it would be a bit clearer and useful to have a named SequentialPalette type and then you do the `as SequentialPalette` inside the map.